#212 - case studies page

AleksandraKozubal opened the pull request:
I created case studies page, single case study page and added changes from general review (that's the last time I'm mixing it to one pr, probably it would be better to add separated pr for those changes).

Case studies page:
![image](https://github.com/blumilksoftware/website/assets/104600942/81f38175-de65-489f-b8a0-6e942bd55beb)
Single case study base page - only presenting name, description and image of certain case study
![image](https://github.com/blumilksoftware/website/assets/104600942/782e9ac7-c373-40e0-bcef-29a6166ae670)

It should close #212 and close #213 - for now we have this one details component, others will be created as needed.


Link to desktop design:
https://xd.adobe.com/view/26f66876-cddd-469c-8b1f-b991dec40e55-73a5/screen/eb45fea3-45c7-4e7e-a91e-d4c1ac710e4a/
and mobile design:
https://xd.adobe.com/view/48e3dd96-96c0-4801-aee3-93a97687467e-1db0/screen/9d2344b4-0163-4f41-941c-7b76aa2e286f/

kamilpiech97 commented on app/Http/Controllers/CaseStudiesController.php:
```diff
@@ -4,18 +4,24 @@
     public function index(Factory $factory): View
     {
-        return $factory->make("case_studies");
+        $caseStudies = CaseStudy::all()->where("published", true);
+
+        return $factory->make("case_studies", compact("caseStudies"));
```
```suggestion
        $caseStudies = CaseStudy::query()->where("published", true)->get();

        return $factory->make("case-studies/$view")
            ->with("caseStudies", $caseStudies);
```

kamilpiech97 replied:
What do u think?

AleksandraKozubal replied:
sure, changed

EwelinaSkrzypacz reviewed (commented):
1. Shadows on design are more visible:

Design:

![image](https://github.com/blumilksoftware/website/assets/56546832/b7f6788a-9878-4c34-93eb-8618798acfe4)

Implementation:

![image](https://github.com/blumilksoftware/website/assets/56546832/b842d251-97c8-4670-b05c-8a4c19dbc70e)

2. Seeders took ages - I'm waiting something like 5 minutes and still seeders don't finish.

3. Mobile view - in design badges, titles and description are centered, but on implementation not:

Design:

![image](https://github.com/blumilksoftware/website/assets/56546832/79f611d3-72ce-4fab-9e90-12cc3f26a23c)


Implementation:

![image](https://github.com/blumilksoftware/website/assets/56546832/7521a45f-eb85-40e7-b1c0-82d42b2e489a)

4. On mobile view, the text "Pozwól nam przekształcić Twoją wizję w rzeczywistość..." is too big compared to design.

5. I see 403 in dev console

![image](https://github.com/blumilksoftware/website/assets/56546832/21af424d-3911-4634-a734-e7601709d640)

6. On design the button "Napisz do nas" is wider.

7. On case study details page, when I hover on button "Wróć do portfolio" it hovers on the next button.

![image](https://github.com/blumilksoftware/website/assets/56546832/ac82a979-f1c0-44a4-9f3d-914ffd65c1a0)

AleksandraKozubal commented on tailwind.config.js:
```diff
@@ -16,8 +16,10 @@ module.exports = {
     extend: {
       colors: {
         gray: {
+          extraLight: '#f5f6fa',
```
yes, changed all to uppercase

AleksandraKozubal commented:
> 1. Shadows on design are more visible
> 
>  2. Seeders took ages - I'm waiting something like 5 minutes and still seeders don't finish.
> 
>  3. Mobile view - in design badges, titles and description are centered, but on implementation not:
> 
>  4. On mobile view, the text "Pozwól nam przekształcić Twoją wizję w rzeczywistość..." is too big compared to design.
> 
>  5. I see 403 in dev console
> 
>  6. On design the button "Napisz do nas" is wider.
> 
>   7. On case study details page, when I hover on button "Wróć do portfolio" it hovers on the next button.
>
1.fixed
2. still working on it
3.fixed
4. changed to smaller font on mobiles
5. Should be better
6. changed
7. fixed

AleksandraKozubal commented:
@EwelinaSkrzypacz I changed factories, now we're not using faker images, so seeder works faster.

kamilpiech97 commented on lang/en/content.php:
```diff
@@ -57,4 +57,14 @@
     "about.section_4.subtitle_1" => "Our company promotes ",
     "about.section_4.subtitle_2" => "work-life balance, ",
     "about.section_4.subtitle_3" => "giving employees the opportunity to develop outside of the professional sphere as well. After hours, our team builds bonds through shared passions such as board games, mountain trips, basketball tournaments, and regular outdoor meetings.",
+    "case_studies.section_1.title_1" => "Our ",
+    "case_studies.section_1.title_2" => "projects",
+    "case_studies.section_1.subtitle_1" => "See what we worked on for our clients.",
+    "case_studies.section_1.subtitle_2" => "Each case study contains a detailed description of the project implementation.",
```
```suggestion
    "case_studies" => [
        "section_1" => [
            "title_2" => "projects",
            "subtitle_1" => "See what we worked on for our clients.",
            "subtitle_2" => "Each case study contains a detailed description of the project implementation."
        ],
    ],
```
maybe that should be more readable?

AleksandraKozubal replied:
formatted
